indico/modules/events/editing/models/file_types.py

```diff
@@ -83,6 +83,17 @@ def __table_args__(cls):
             lazy=True
         )
     )
+    review_conditions = db.relationship(
+        'EditingReviewCondition',
+        secondary='event_editing.review_condition_file_types',
+        collection_class=set,
+        lazy=True,
+        backref=db.backref(
+            'file_types',
+            collection_class=set,
+            lazy=True
```

Comment:
The number of queries on the file type management page increases with each condition added. Not a big problem since we don't have *that*many conditions, but better make this one `lazy=False` (if you end up moving the relationship to the other side, this will then be in the "outer" part, not inside the backref)